components/script_traits/lib.rs

```diff
@@ -184,6 +184,9 @@ pub struct LoadData {
     pub srcdoc: String,
     /// The inherited context is Secure, None if not inherited
     pub inherited_secure_context: Option<bool>,
+
+    /// Servo internal: if crash details are present, trigger a crash error page with these details
```

Comment:
Ditto.

Reply:
fixed in c85c102311a8914f8a34f0fb6932be6b2f163382